clang/lib/Sema/SemaCodeComplete.cpp

```diff
@@ -6749,6 +6749,55 @@ void SemaCodeCompletion::CodeCompleteInitializer(Scope *S, Decl *D) {
   CodeCompleteExpression(S, Data);
 }
 
+void SemaCodeCompletion::CodeCompleteIfConst(Scope *S,
+                                             bool AfterExclaim) const {
+  ResultBuilder Results(SemaRef, CodeCompleter->getAllocator(),
+                        CodeCompleter->getCodeCompletionTUInfo(),
+                        CodeCompletionContext::CCC_Other);
```

Comment:
Looking at the similar function `CodeCompleteAfterIf`, I think `mapCodeCompletionContext(SemaRef, PCC_Statement)` would be more appropriate here than `CodeCompletionContext::CCC_Other`

Reply:
![image](https://github.com/user-attachments/assets/4ba0669a-8495-46a0-9b48-9ff9bde4adca)
I think they are different. In `CodeCompleteAfterIf`, it's valid to provide a statement. While in the new added `CodeCompleteKeywordAfterIf ` only two keywords are appropriate.